### DISCUSSION TITLE
[Bug] 使用one-api挂载nvdia的模型nvidia/nemotron-4-340b-instruct时出现报错，而fastgpt可以正常使用

### DISCUSSION THREAD

**Discussion by liuzj288** [lobehub/lobehub#3178] [General Question] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

1.2.14

### 💻 系统环境

Other Linux

### 🌐 浏览器

Chrome

### 🐛 问题描述

1.one-api挂载没问题
![image](https://github.com/lobehub/lobe-chat/assets/13833404/f1ec60cf-32e8-442d-b83c-4a198a2ba71e)
2.meta/llama3-70b-instruct使用没有问题，nvidia/nemotron-4-340b-instruct和google/gemma-2-9b-it都报错
![image](https://github.com/lobehub/lobe-chat/assets/13833404/f565b6da-cbd8-48e7-8e01-2a97b835229e)
3.在fastgpt中可以正常使用
![image](https://github.com/lobehub/lobe-chat/assets/13833404/8c7a3184-e276-4ee3-ab65-396b76b2c86b)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
你应该去问下 one-api 422 是什么错误吧？

[a comment by an automated account is not shown]

**Comment by liuzj288** [comment]
> 你应该去问下 one-api 422 是什么错误吧？

类似这个问题https://github.com/songquanpeng/one-api/issues/1290，看了下回答说是
![image](https://github.com/lobehub/lobe-chat/assets/13833404/cb80014a-cd88-40b9-803e-aea065aa8038)

这个有能设置的地方吗？

**Comment by arvinxx** [comment, marked as answer]
没有， openai 是支持的，不可能为了三方代理的接口去掉这两个配置。

[a comment by an automated account is not shown]
